[CI][windows] tests.metrics.breakdown_tests failed with python 3.7 in Windows

`windows-3.7-none / test_multiple_transactions – tests.metrics.breakdown_tests` failed with the below stacktrace

```
Error
assert 9999999 == 10000000
Stacktrace
elasticapm_client = <tests.fixtures.TempStoreClient object at 0x000001FF75350E08>
    def test_multiple_transactions(elasticapm_client):
        for i in (1, 2):
            elasticapm_client.begin_transaction("request")
            with elasticapm.capture_span("test", duration=5):
                pass
            elasticapm_client.end_transaction("test", "OK", duration=10)
    
        breakdown = elasticapm_client._metrics.get_metricset("elasticapm.metrics.sets.breakdown.BreakdownMetricSet")
        data = list(breakdown.collect())
        asserts = 0
        for elem in data:
            if "transaction.breakdown.count" in elem["samples"]:
                assert elem["samples"]["transaction.breakdown.count"]["value"] == 2
                assert elem["transaction"] == {"name": "test", "type": "request"}
                asserts += 1
            elif "span.self_time.sum.us" in elem["samples"]:
                if elem["span"] == {"type": "app", "subtype": ""}:
                    assert elem["transaction"] == {"name": "test", "type": "request"}
                    # precision lost due to float arithmetic
>                   assert elem["samples"]["span.self_time.sum.us"]["value"] == 10000000
E                   assert 9999999 == 10000000
tests\metrics\breakdown_tests.py:231: AssertionError
```

See [here](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-python%2Fapm-agent-python-mbp/detail/PR-746/24/tests)

Already discussed with @beniwohli and suggested to use some assertion a bit less strict

--- a/tests/metrics/breakdown_tests.py
+++ b/tests/metrics/breakdown_tests.py
@@ -228,12 +228,12 @@ def test_multiple_transactions(elasticapm_client):
             if elem["span"] == {"type": "app", "subtype": ""}:
                 assert elem["transaction"] == {"name": "test", "type": "request"}
                 # precision lost due to float arithmetic
-                assert elem["samples"]["span.self_time.sum.us"]["value"] == 10000000
+                assert 9999999 <= elem["samples"]["span.self_time.sum.us"]["value"] <= 10000000
                 assert elem["samples"]["span.self_time.count"]["value"] == 2
                 asserts += 1
             elif elem["span"] == {"type": "code", "subtype": "custom"}:
                 assert elem["transaction"] == {"name": "test", "type": "request"}
-                assert elem["samples"]["span.self_time.sum.us"]["value"] == 10000000
+                assert 9999999 <= elem["samples"]["span.self_time.sum.us"]["value"] <= 10000000
                 assert elem["samples"]["span.self_time.count"]["value"] == 2
                 asserts += 1
     assert asserts == 3
@@ -243,5 +243,5 @@ def test_multiple_transactions(elasticapm_client):
     )
     transaction_data = list(transaction_metrics.collect())
     assert len(transaction_data) == 1
-    assert transaction_data[0]["samples"]["transaction.duration.sum.us"]["value"] == 20000000
+    assert 19999999 <= transaction_data[0]["samples"]["transaction.duration.sum.us"]["value"] <= 20000000
     assert transaction_data[0]["samples"]["transaction.duration.count"]["value"] == 2

--- a/tests/requirements/requirements-psutil-5.0.txt
+++ b/tests/requirements/requirements-psutil-5.0.txt
@@ -1,2 +1,2 @@
-psutil>=5.0.0,<6.0
+psutil==5.0.0
 -r requirements-base.txt

